{
	"id": "RLSA-2022:7323",
	"advisories": [
		{
			"content": {
				"id": "RLSA-2022:7323",
				"title": "Moderate: python3.9 security update",
				"description": "Python is an interpreted, interactive, object-oriented programming language, which includes modules, classes, exceptions, very high level dynamic data types and dynamic typing. Python supports interfaces to many system calls and libraries, as well as to various windowing systems.\n\nSecurity Fix(es):\n\n* python: int() type in PyLong_FromString() does not limit amount of digits converting text to int leading to DoS (CVE-2020-10735)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
				"severity": [
					{
						"type": "vendor",
						"source": "errata.rockylinux.org",
						"vendor": "MODERATE"
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10735"
					},
					{
						"source": "Red Hat",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1834423"
					},
					{
						"source": "errata.rockylinux.org",
						"url": "https://errata.rockylinux.org/RLSA-2022:7323"
					}
				],
				"published": "2022-11-02T13:53:37Z"
			},
			"segments": [
				{
					"ecosystem": "rocky:9"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-10735",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "MITRE",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
							"base_score": 7.5,
							"base_severity": "HIGH",
							"temporal_score": 7.5,
							"temporal_severity": "HIGH",
							"environmental_score": 7.5,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "MITRE",
						"cwe": [
							"CWE-704->CWE-400"
						]
					}
				],
				"references": [
					{
						"source": "MITRE",
						"url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10735"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "rocky:9"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "rocky:9",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python-unversioned-command",
										"architectures": [
											"noarch"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3",
										"architectures": [
											"aarch64",
											"i686",
											"ppc64le",
											"s390x",
											"x86_64"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3-debug",
										"architectures": [
											"aarch64",
											"i686",
											"ppc64le",
											"s390x",
											"x86_64"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3-devel",
										"architectures": [
											"aarch64",
											"i686",
											"ppc64le",
											"s390x",
											"x86_64"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3-idle",
										"architectures": [
											"aarch64",
											"i686",
											"ppc64le",
											"s390x",
											"x86_64"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3-libs",
										"architectures": [
											"aarch64",
											"i686",
											"ppc64le",
											"s390x",
											"x86_64"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3-test",
										"architectures": [
											"aarch64",
											"i686",
											"ppc64le",
											"s390x",
											"x86_64"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3-tkinter",
										"architectures": [
											"aarch64",
											"i686",
											"ppc64le",
											"s390x",
											"x86_64"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							},
							{
								"type": "version",
								"version": {
									"vulnerable": true,
									"fix_status": {
										"class": "fixed"
									},
									"package": {
										"name": "python3.9-debuginfo",
										"architectures": [
											"aarch64",
											"ppc64le",
											"s390x",
											"x86_64"
										]
									},
									"affected": {
										"type": "rpm",
										"range": [
											{
												"lt": "0:3.9.10-3.el9_0"
											}
										],
										"fixed": [
											"0:3.9.10-3.el9_0"
										]
									}
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "rocky-errata",
		"raws": [
			"vuls-data-raw-rocky-errata/2022/RLSA-2022:7323.json"
		]
	}
}
